web/src/ui_init.js

```diff
@@ -424,6 +425,12 @@ export function initialize_everything(state_data) {
         initialization function for the corresponding module.
     */
 
+    document.addEventListener("copy", (ev) => {
+        if (copy_messages.copy_handler(ev)) {
+            ev.preventDefault();
+        }
+    });
```

Comment:
This should go in a `copy_messages.initialize` function and shouldn’t be so early in the sequence.